__tests__/fade.test.js

```diff
@@ -218,3 +218,14 @@ test('When the pauseOnHover prop is true and the mouse hovers the container the
     }
   );
 });
+
+test('cssClass prop exists on element when it is passed', () => {
+  const { container } = renderFade({
+    ...fadeProperties,
+    cssClass: 'myStyle'
+  });
+  let elementToStyle = container.querySelectorAll(
+    '.react-slideshow-fade-wrapper'
+  );
+  expect(elementToStyle.length).toBe(1);
```

Comment:
This assertion doesn't actually check if `myStyle` was added to the `.react-slideshow-fade-wrapper`
You should use something like this 
```js
const wrapper = container.querySelectorAll(
    '.react-slideshow-fade-wrapper'
  );
  expect(wrapper.classList).toContain('myStyle');
```

Reply:
Thanks!, Will update it.